src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -24,10 +24,13 @@ internal class MonoProxy : DevToolsProxy
         private Dictionary<SessionId, ExecutionContext> contexts = new Dictionary<SessionId, ExecutionContext>();
         private const string sPauseOnUncaught = "pause_on_uncaught";
         private const string sPauseOnCaught = "pause_on_caught";
+        // index of the runtime in a same JS page/process
+        public int RuntimeId { get; private set; }
```

Comment:
Can this ever change after the object is instantiated? Is not, then make this `init` property.